### PR TITLE
Updates to troubleshooting

### PR DESCRIPTION
**Pull request by natefoo**
There's some stuff to clean up in here but I'm making a PR so that I don't forget about it.

### CODE REVIEW

File: topics/admin/tutorials/troubleshooting/slides.html [hunk 1]
```diff
@@ -151,22 +304,47 @@
 ---
 # Tool errors - stderr
 
+**Galaxy considers consider *any* output to standard error (stderr) to be an error when no tool profile version is set by a tool.**
```
Comment (bgruening) [inline topics/admin/tutorials/troubleshooting/slides.html]: s/consider//

File: topics/admin/tutorials/troubleshooting/slides.html [hunk 2]
```diff
@@ -195,89 +374,110 @@
 - Increase the amount of memory available to the job
   - Stop other jobs
   - Add memory
-  - Cluster (advanced)
+  - Cluster (and reserve memory for jobs)
+  - Use job resubmission (Slurm only) to automatically rerun with a larger memory allocation
```
Comment (bgruening) [inline topics/admin/tutorials/troubleshooting/slides.html]: I think we are using this as well with condor. @erasche?

**Comment by erasche** [comment]
Yeah resubmission isn't specific. Should include an exercise for that
somewhere hmm

Op za 2 feb. 2019 09:32 schreef Björn Grüning <notifications@github.com:

File: topics/admin/tutorials/troubleshooting/slides.html (hunk 1 above)
Comment (erasche) [inline topics/admin/tutorials/troubleshooting/slides.html]:
```suggestion
**Galaxy considers *any* output to standard error (stderr) to be an error when no tool profile version is set by a tool.**
```

File: topics/admin/tutorials/troubleshooting/slides.html (hunk 2 above)
Comment (erasche) [inline topics/admin/tutorials/troubleshooting/slides.html]:
```suggestion
  - Use job resubmission to automatically rerun with a larger memory allocation
```

**Pull request merged**